### ISSUE TITLE
Dom manipulation in jailed

### ISSUE THREAD

**Issue by AKNiazi** [asvd/jailed#48]
Is there a way to manipulate parent dom in jailed iframe.

**Comment by Guard13007** [comment]
Yes, by making a function that does this manipulation accessible to the jailed script. The first example shows how to do this with `alert()`.
